pkg/pool-weighted/contracts/managed/ManagedPool.sol

```diff
@@ -229,6 +248,21 @@ contract ManagedPool is ManagedPoolSettings {
         // amountsIn are amounts entering the Pool, so we round up.
         _downscaleUpArray(amountsIn, scalingFactors);
 
+        // BasePool will mint bptAmountOut for the sender: we then also mint the remaining BPT to make up the total
```

Comment:
who is this base pool you speak of

Reply:
https://github.com/balancer-labs/balancer-v2-monorepo/blob/9458439f6f9c4918e33bb6f05ce07ae493151400/pkg/pool-weighted/contracts/managed/vendor/BasePool.sol#L223-L224

Reply:
Yeah I have no idea what i was referring to